docs/05-concepts/02-protocol.md

```diff
@@ -16,6 +16,8 @@ fields:
   employees: List<Employee>
 ```
 
+Supported types are `bool`, `int`, `double`, `String`, `DateTime`, `ByteData`, and other serializable classes. You can also use `List`s and `Map`s of the supported types, just make sure to specify the types. Null safety is supported. The keys of `Map` must be non-nullable `String`s. Once your classes are generated, you can use them as parameters or return types to endpoint methods.
```

Comment:
Question: are the types listed in many places? Maybe we should have then defined in a single place and reference it?

Reply:
No it should be only here I believe, they where inside the serialisation page before.